tests/test_builder_containerbuild.py

```diff
@@ -54,6 +55,60 @@ def mock_incremental_upload(session, fname, fd, uploadpath, logger=None):
 
 
 class TestBuilder(object):
+    @pytest.mark.parametrize(('flags', 'valid'), [
+        ({'http://internal.status.redhat.com/': ['osbs-deployments-allowed']}, True),
```

Comment:
Can we use some generic domains?

Reply:
I was hoping to be able actually test the call, not just mock it.